docs/lib/restapi/fragments/js/authz.md

```diff
@@ -13,13 +13,28 @@ Amplify.configure({
           return { Authorization : 'token' } 
           // Alternatively, with Cognito User Pools use this:
           // return { Authorization: `Bearer ${(await Auth.currentSession()).getAccessToken().getJwtToken()}` }
+          // return { Authorization: `Bearer ${(await Auth.currentSession()).getIdToken().getJwtToken()}` }
         }
       }
     ]
   }
 });
 ```
 
+## Note related to use Access Token or ID Token
+
+The ID Token contains claims about the identity of the authenticated user such as name, email, and phone_number. It could have custom claims as well (For example using [Amplify CLI](https://docs.amplify.aws/cli/usage/lambda-triggers#override-id-token-claims) ). On Amplify Auth category you can retrieve the Id Token using: 
```

Comment:
```suggestion
The ID Token contains claims about the identity of the authenticated user such as name, email, and phone_number. It could have custom claims as well, for example using [Amplify CLI](https://docs.amplify.aws/cli/usage/lambda-triggers#override-id-token-claims). On the Amplify Authentication category you can retrieve the Id Token using: 
```

Reply:
```suggestion
The ID Token contains claims about the identity of the authenticated user such as name, email, and phone_number. It could have custom claims as well, for example using [Amplify CLI](https://docs.amplify.aws/cli/usage/lambda-triggers#override-id-token-claims). On the Amplify Authentication category you can retrieve the Id Token using: 
```